python/ray/tests/test_client_library_integration.py

```diff
@@ -0,0 +1,26 @@
+import sys
+
+import pytest
+
+from ray.rllib.examples import rock_paper_scissors_multiagent
+from ray.util.client.ray_client_helpers import ray_start_client_server
+from ray._private.client_mode_hook import _explicitly_enable_client_mode,\
+    client_mode_should_convert
+
+
+@pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows.")
+def test_rllib_integration(ray_start_regular_shared):
+    with ray_start_client_server():
+        # Confirming the behavior of this context manager.
+        # (Client mode hook not yet enabled.)
+        assert not client_mode_should_convert()
+        # Need to enable this for client APIs to be used.
+        _explicitly_enable_client_mode()
```

Comment:
Thanks for making it very clear that we are using client mode!

Reply:
The thing to note is that the context manager does /not/ enable client mode -- it sets a flag
`_in_client_test` to prevent `ray.connect` from switching on client mode.